scripts/metalsmith.js

```diff
@@ -184,6 +184,14 @@ exports.metalsmith = function() {
           'hardware-projects'
         ]
       },
+      workshops: {
```

Comment:
There are two workshops sections. One here and one on line 205. I don't think you want both.